types/office-js-preview/index.d.ts

```diff
@@ -11729,7 +11731,9 @@ declare namespace Office {
          */
         setSignatureAsync(data: string, options?: Office.AsyncContextOptions & CoercionTypeOptions, callback?: (asyncResult: Office.AsyncResult<void>) => void): void;
         /**
-         * Adds or replaces the signature of the email body.
+         * Adds or replaces the signature of the item body.
```

Comment:
This appears to be on `dsiableClientSignatureAsync` rather than `setSignatureAsync`.

Reply:
@AlexJerabek This message is in the description for setSignatureAsync so this is correct. Notice the diff in line numbers.

Reply:
Awesome! I wish that didn't look so terrible in GitHub...